config/haproxy/haproxy.cfg

```diff
@@ -11,6 +11,8 @@ defaults
         timeout client 120000
         timeout server 120000
 
+errorfile 503 /etc/caasp/haproxy/errors/503_database_not_ready.html.http
```

Comment:
Maybe we can use a rename of the file... because it's not that the database is not ready, it might be a myriad of different reasons.

Aside from that, could we move this inside the `defaults` block?

Reply:
renamed and added the errorfile in the `listen velum` block